### PR TITLE
DAOS-7113: Fix rebuild/delete_objects.py variant 3 - Error data mismatch

### PR DESCRIPTION
**Pull request by dinghwah**
DAOS-7113: Fix rebuild/delete_objects.py variant 3 - Error data mismatch.
modified:   delete_objects.py
modified:   ../util/test_utils_container.py
Skip-unit-tests: true
Skip-nlt: true
Quick-Functional: true
Test-tag: rebuild_tests
Signed-off-by: Ding Ho <ding-hwa.ho@intel.com>

### CODE REVIEW

**Review by shimizukko — APPROVED** [review]

**Comment by dinghwah** [comment]
There are failures on unrelated to this rebuild delete_object tests, isolating the issue with different PR.

**Comment by dinghwah** [comment]
There are failures on cascading test, due to an existing JIRA bug, isolate the PR to individual rebuild tests.

**Comment by dinghwah** [comment]
Due to CentOS7 build rpm failed

**Review by shimizukko — DISMISSED** [review]

File: src/tests/ftest/rebuild/delete_objects.py [hunk 1]
```diff
@@ -90,7 +86,10 @@ def test_rebuild_delete_objects(self):
         Use Cases:
             foo
 
-        :avocado: tags=all,large,full_regression,rebuild,rebuilddeleteobject
+        :avocado: tags=all,daily_regression
+        :avocado: tags=vm,large
+        :avocado: tags=rebuild
+        :avocado: tags=pool,rebuild_tests,rebuilddeleteobject
```
Comment (saurabhtandan) [inline src/tests/ftest/rebuild/delete_objects.py:92]: can you please change `rebuild_tests` to `deleteobjects` as something similar is already being used under rebuild/rebuild_tests.py

File: src/tests/ftest/rebuild/delete_objects.py [hunk 2]
```diff
@@ -107,7 +106,10 @@ def test_rebuild_delete_records(self):
         Use Cases:
             foo
 
-        :avocado: tags=all,large,full_regression,rebuild,rebuilddeleterecord
+        :avocado: tags=all,daily_regression
+        :avocado: tags=vm,large
+        :avocado: tags=rebuild
+        :avocado: tags=pool,rebuild_tests,rebuilddeletrecord
```
Comment (saurabhtandan) [inline src/tests/ftest/rebuild/delete_objects.py:112]: Same as above.

**Review by saurabhtandan — CHANGES_REQUESTED** [review]
After the change you will need to run the PR with `deleteobjects` tag along with all other pr tests as this PR makes changes to utils file.

**Review by shimizukko — DISMISSED** [review]

**Comment by dinghwah** [comment]
All test passed on the testcases on test-Tag.

File: src/tests/ftest/rebuild/delete_objects.py [hunk 3]
```diff
@@ -90,7 +86,9 @@ def test_rebuild_delete_objects(self):
         Use Cases:
             foo
 
-        :avocado: tags=all,large,full_regression,rebuild,rebuilddeleteobject
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm,large
```
Comment (ravalsam) [inline src/tests/ftest/rebuild/delete_objects.py:90]: large tag is needed?

**Review by ravalsam — COMMENTED** [review]
This code is updating container utility so to land this patch need to run all standard CI tests, isn't it?

**Review by dinghwah — COMMENTED** [review]
Thanks for the comment.
I have updated the test-tag after discussed team.
Also added entire rebuild testcases into the the is PR.
Please see next commit.

**Review by shimizukko — APPROVED** [review]

**Comment by dinghwah** [comment]
Resubmitted PR with all the rebuild tests.
3 tests failed are due to known issue DAOS-7169.
All rest tests on the rebuild are passed.

**Review by ravalsam — APPROVED** [review]

**Review by mjean308 — APPROVED** [review]
lgtm

**Review by shimizukko — APPROVED** [review]

**Comment by sylviachanoiyee** [comment]
Can you run all the functional tests? There is change to the utils file. Thanks!

**Comment by dinghwah** [comment]
Fix has been merge with PR# 5171, Close this PR to avoid conflict.

**Pull request closed without merging**